pkg/collector/corechecks/orchestrator/pod/stub.go

```diff
@@ -0,0 +1,7 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package pod is used for the orchestrator pod check
+package pod
```

Comment:
what's the goal of the 2 stub files?

Reply:
Because we use build tags https://github.com/DataDog/datadog-agent/blob/86b53dda83fd6f0fe960e8c84f1106bb7b0dd6ea/cmd/agent/subcommands/run/command.go#L104 will fail if a build is kicked off without our build tags. 

I forget which architecture this had happened to in the past, but this stub file allows the package to be created and just do nothing. You can see the pattern used quite a lot in other folders.